test/test_cstdfloat.cpp

```diff
@@ -304,6 +304,8 @@ namespace test_cstdfloat
       }
       s = 8 * std::numeric_limits<T>::epsilon();
       val = 2.5;
+
+      #ifndef __STRICT_ANSI__ 
```

Comment:
We have a macro `BOOST_MATH_LARGEST_FLOAT_C(val)` declared in boost/math/tools/big_constant.hpp which declares `val` as the largest supported literal.

But there's another issue here, the function test() appears to be never called?  @ckormanyos ?